### PR TITLE
Add Banner size API and banner position on editor

### PR DESCRIPTION
**Pull request by rodrilemos**

### CODE REVIEW

File: Assets/ScaleMonk Ads/Editor/AdnetsWindow/AdsProvidersHelper.cs [hunk 1]
```diff
@@ -16,8 +16,8 @@ namespace ScaleMonk.Ads
 {
     public class AdsProvidersHelper
     {
-        const string iosAdsVersion = "1.1.0";
-        const string androidAdsVersion = "1.0.23";
+        const string iosAdsVersion = "1.2.0";
+        const string androidAdsVersion = "1.1.0-rc.11";
```
Comment (nravasi) [inline Assets/ScaleMonk Ads/Editor/AdnetsWindow/AdsProvidersHelper.cs:20]: no subas esta version rc a master

File: Assets/ScaleMonk Ads/Source/ScaleMonkAds.cs [hunk 2]
```diff
@@ -161,21 +175,21 @@ public void ShowRewarded(string tag)
...
         /// <summary>
         /// Displays a banner ad.
         ///
         /// If the display was successful, the event `BannerCompletedDisplayedEvent` will be called.
         /// Otherwise, the event `BannerFailedDisplayedEvent` will be called.
         /// </summary>
-        /// <param name="tag">The game tag from where the ad will be displayed (like menu or store).</param>
+        /// <param name="bannerSize">The bannerSize of the Ad.</param>
         /// <param name="bannerPosition">The bannerPosition where the ad will be displayed.</param>
-        public void ShowBanner(string tag, BannerPosition bannerPosition)
```
Comment (nravasi) [inline Assets/ScaleMonk Ads/Source/ScaleMonkAds.cs:175]: No hay un metodo que reciba tag, position y size?

**Review by nravasi — CHANGES_REQUESTED** [review]

File: Assets/ScaleMonk Ads/Editor/AdnetsWindow/AdsProvidersHelper.cs (hunk 1 above)
Comment (fgaule) [reply Assets/ScaleMonk Ads/Editor/AdnetsWindow/AdsProvidersHelper.cs:20]: You can set android verison 1.1.0, it has been released yesterday

Comment (rodrilemos) [reply Assets/ScaleMonk Ads/Editor/AdnetsWindow/AdsProvidersHelper.cs]: Fixed

File: Assets/ScaleMonk Ads/Source/ScaleMonkAds.cs (hunk 2 above)
Comment (rodrilemos) [reply Assets/ScaleMonk Ads/Source/ScaleMonkAds.cs:175]: Yeah, my bad. Done

**Review by nravasi — APPROVED** [review]

**Pull request merged**